src/cli/hook-utils/manifest.spec.js

```diff
@@ -0,0 +1,61 @@
+require('mocha');
+const { expect } = require('chai');
+const { vol, createFsFromVolume } = require('memfs');
+const { unionMerge, hasManifest, find } = require('./manifest');
+
+describe('Slack CLI Script Hooks: get-manifest utilities', () => {
```

Comment:
I have added some initial unit tests to get things kicked off, starting with the low-level hook utilities.